extensions/32-bit-address-space/README.md

```diff
@@ -0,0 +1,58 @@
+# General Design
+
+**Extension State: Under Development**
+**Requires: CP1.14**
+**CP Bit: CP1.16**
+
+# Overview
+
+This extension adds a new concept of _modes_ to the ISA. Modes are used to control things about the processor
+state which are binary-incompatible - that is, binary code written for a mode other than the mode the
+processor is actually in will almost certainly behave incorrectly.
+
+In particular, this extension focuses on the _Real 32-bit address mode_.
```

Comment:
Maybe we should define what "real" means in this context for anyone who isn't familiar with the terminology?

Reply:
I think not. In a vacuum it's just a name that doesn't have to mean anything. It's only meaningful when contrasted against virtual or segmented memory.

I tried describing it, realized that that essentially requires saying "in contrast to virtual memory" (because why would an address _not_ refer to that location in physical memory?) and then gave up. I think the virtual memory extension is the place to point out that something clever can happen.

Reply:
That's a fair point